src/vma/proto/mem_buf_desc.h

```diff
@@ -123,7 +124,8 @@ class mem_buf_desc_t {
 			size_t		sz_payload;
 
 			struct timespec sw_timestamp;
```

Comment:
why not remove the sw_tiemstamp? 
It is calculated in last line before reporting to user. I think we don't need it anymore in VMA's mem_buf_desc_t.